### PR TITLE
Document the camera recognition feature

### PR DESCRIPTION
**Pull request by fabienrohrer**
Fix https://github.com/omichel/webots-doc/issues/873

Preview: https://www.cyberbotics.com/doc/reference/camera?version=enhancement-camera-recognition-documentation#overlay-image

### CODE REVIEW

**Comment by fabienrohrer** [comment]
👍

**Comment by DavidMansolino** [comment]
👀

File: reference/camera.md [hunk 1]
```diff
@@ -186,6 +186,15 @@ It is also possible to show the camera image in an external window by double-cli
 After doing it, the overlay disappears and a new window pops up.
 Then, after closing the window, the overlay will be automatically restored.
 
+When the [recognition mode](#wb_camera_has_recognition) is enabled and the simulator is paused, yellow rectangles surround the recognized objects.
```
Comment (DavidMansolino) [inline reference/camera.md]: The rectangle are always present even in non-pause mode.

File: reference/camera.md [hunk 2]
```diff
@@ -186,6 +186,15 @@ It is also possible to show the camera image in an external window by double-cli
...
+If the mouse cursor is over one of these yellow rectangles, then a complete description of the recognized object is displayed in red, as shown in [the following figure](#display-of-a-recognized-object).
```
Comment (DavidMansolino) [inline reference/camera.md]: In this case it need to be in pause mode.

File: reference/camera.md (hunk 1 above)
Comment (DavidMansolino) [reply reference/camera.md]: I would remove the 'yellow' world as by default they are red (see 'frameColor' in https://www.cyberbotics.com/doc/reference/recognition#field-summary).

**Comment by fabienrohrer** [comment]
Thank you.

👍

**Pull request merged**